extensions/amp-story/1.0/amp-story-open-page-attachment.css

```diff
@@ -246,6 +246,10 @@
   position: relative !important;
 }
 
+.i-amphtml-story-outlink-page-attachment[active] {
```

Comment:
Do you need the [active]? If it's for CSS specificity, I think we could do `.i-amphtml-story-page-open-attachment.i-amphtml-story-outlink-page-attachment` so the background never renders at all, even on inactive pages

Reply:
Revised this to remove the background on all CTA variations in the experiment.